packages/aws-cdk-lib/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -1530,6 +1550,9 @@ export class AutoScalingGroup extends AutoScalingGroupBase implements
 
     const asgProps: CfnAutoScalingGroupProps = {
       autoScalingGroupName: this.physicalName,
+      availabilityZoneDistribution: props.azCapacityDistributionStrategy
+        ? { capacityDistributionStrategy: props.azCapacityDistributionStrategy }
+        : { capacityDistributionStrategy: CapacityDistributionStrategy.BALANCED_BEST_EFFORT },
```

Comment:
Thank you for the fix.
But I think default value should be `undefined`.
The `@default` should describe the behavior at that time.

When the default value is explicitly set, it causes a breaking change.
This is because before and after this PR merged, the synth result changes when `props.azCapacityDistributionStrategy` is omitted.

Reply:
@mazyu36 

thank you!

I see, I understand.

So, I will also omit the description of default in JSDoc, is that correct?  
(Because there was no mention of default values in the CloudFormation documentation...)  
https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/aws-properties-autoscaling-autoscalinggroup-availabilityzonedistribution.html

Reply:
For optional properties, `@default` is necessary. 

Also, it's common that default value behaviors are not documented in the CFn documentation.
In such cases, I often verify the default value by deploying without specifying the value.

When I tried deploying it without `availabilityZoneDistribution` property, the value became `Balanced best effort`, so I think this is the default value.

![image](https://github.com/user-attachments/assets/d6cedcb7-2e62-4d74-9580-af3871860a8d)

Reply:
Thank you for the details.  
I have corrected it with the following commit!

https://github.com/aws/aws-cdk/pull/32100/commits/ea1e756446da8768dc4cd70b5f885811315b9ed8